shared/common-adapters/switch.stories.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import React from 'react'
+import Box, {Box2} from './box'
+import ClickableBox from './clickable-box'
+import Switch from './switch'
+import * as Sb from '../stories/storybook'
+
+const Kb = {
+  Box,
+  Box2,
+  ClickableBox,
+  Switch,
+}
+
+class TestWrapper extends React.PureComponent<
+  {
+    align: 'left' | 'right',
+    color: 'blue' | 'green',
+    disabled?: ?boolean,
+    initialEnabled: boolean,
+    label: string,
+  },
+  {on: boolean}
+> {
+  state = {on: this.props.initialEnabled}
+  _onClick = () => this.setState(({on}) => ({on: !on}))
+  render() {
+    return (
+      <Kb.Switch
+        align={this.props.align}
+        color={this.props.color}
+        disabled={this.props.disabled}
+        label={this.props.label}
+        on={this.state.on}
+        onClick={this._onClick}
+        style={{width: 320}}
+      />
+    )
+  }
+}
+
+const load = () => {
+  Sb.storiesOf('Common', module).add('Switch', () => (
+    <Kb.Box2 direction="vertical" gap="large" gapStart={true} centerChildren={true} fullWidth={true}>
+      {['left', 'right'].map(align =>
+        ['blue', 'green'].map(color =>
+          [true, false, 'disabled'].map(i => {
+            const initialEnabled = !!i
+            const label = `${align} aligned; ${initialEnabled ? 'enabled' : 'disabled'}; ${color}${
```

Comment:
two `disabled`s in the labels is confusing
```suggestion
            const label = `${align} aligned; ${initialEnabled ? 'on' : 'off'}; ${color}${
```